### ISSUE TITLE
Provide better error message when `profile.release.strip = "symbols"`

### ISSUE THREAD

**Issue by YurBoiRene** [iai-callgrind/iai-callgrind#92]
## The Problem

If you have `profile.release.strip = "symbols"` in your Cargo.toml (a common binary size optimization) then `iai-callgrind` will attempt your benchmarks as normal, only erroring on parsing the callgrind output with a cryptic error message:

```
my_benchmark::bench_fibonacci_group::bench_fibonacci short:10
iai_callgrind_runner: Error: Error parsing file '/home/rene/Documents/Java/callgrind/callgrind-test/target/iai/callgrind-test/my_benchmark/bench_fibonacci_group/bench_fibonacci.short/callgrind.bench_fibonacci.short.out': Sentinel 'iai_callgrind::bench::' not found
error: bench failed, to rerun pass `--bench my_benchmark`

Caused by:
  process didn't exit successfully: `/home/rene/Documents/Java/callgrind/callgrind-test/target/release/deps/my_benchmark-14c0441f599a108e --bench` (exit status: 1)
```

The contents of `callgrind.bench_fibonacci.short.out`:

```
# callgrind format
version: 1
creator: callgrind-3.22.0
pid: 15487
cmd:  /home/rene/Documents/Java/callgrind/callgrind-test/target/release/deps/my_benchmark-14c0441f599a108e --iai-run bench_fibonacci_group 0 0 my_benchmark::bench_fibonacci_group::bench_fibonacci
part: 1


desc: I1 cache: 32768 B, 64 B, 8-way associative
desc: D1 cache: 32768 B, 64 B, 8-way associative
desc: LL cache: 8388608 B, 64 B, 16-way associative

desc: Timerange: Basic block 0 - 71579
desc: Trigger: Program termination

positions: line
events: Ir Dr Dw I1mr D1mr D1mw ILmr DLmr DLmw
summary: 0


totals: 0
```

I am unfamiliar with this packages internals but from my understanding callgrind puts a debug symbol in its output which iai looks for. However, if those Debug symbols are stripped then it is not present in the output -> Error.

## The Fix

A simple fix for the user is to add `profile.bench.strip = "none"` which will override the release symbol stripping. However, this fix is not apparent at all from the current `iai-callgrind` error message.

## TL;DR
if this happens, catch it and print out "make sure you add`profile.bench.strip = "none"`"

I'd be happy to implement and make an MR if this sounds okay.

**Comment by gamma0987** [comment]
Thanks for reporting this.

Yeah, this error message should be improved. If you want to start a pr, go ahead. This cargo setting also needs to be documented in the README. Maybe together with `debug = true`.

**Comment by gamma0987** [comment]
I updated the README and the error message. I hope it's clearer now.
